imports/plugins/included/shippo/register.js

```diff
@@ -0,0 +1,30 @@
+import { Reaction } from "/server/api";
+
+Reaction.registerPackage({
+  label: "Shippo",
+  name: "reaction-shippo",
+  icon: "fa fa-plane",
+  autoEnable: true,
+  settings: {
+    api_key: ""
+  },
+  registry: [{
+    provides: "dashboard",
+    label: "Shippo",
+    description: "Shippo service",
+    icon: "fa fa-at",
+    priority: 3,
+    container: "connect",
+    permissions: [{
+      label: "Shippo",
+      permission: "dashboard/shippo"
+    }]
+  }, {
+    label: "Shippo",
+    route: "/dashboard/shippo",
+    provides: "settings",
+    container: "connection",
+    template: "shippoSettings"
+  }
```

Comment:
Don't we need something like this block? (i.e. we need a checkout template as well)

```
    {
      template: "flatRateCheckoutShipping",
      name: "shipping/flatRates",
      provides: "shippingMethod"
    }
```